### ISSUE TITLE
Failed test\regression\int_pruf_plant_analysis.py F

### ISSUE THREAD

**Issue by charlie9578** [NREL/OpenOA#154]
I'm not sure whats causing this one, but in general the test\regression\int_pruf_plant_analysis.py test is failing for me. An example is below. The y-array values vary slightly from test-to-test, which suggests to me this might be a statistical problem.

```
self = <test.regression.int_pruf_plant_analysis.TestPandasPrufPlantAnalysis testMethod=test_plant_analysis>
s =       aep_GWh  avail_pct  curt_pct  lt_por_ratio        r2       mse  n_points       iav
0   12.975444   0.011491  0.0...  0.005043      24.0  0.106800
29  12.143577   0.012278  0.000683      1.016500  0.839426  0.020366      24.0  0.058938

    def check_simulation_results_lin_monthly(self, s):
        # Make sure AEP results are consistent to one decimal place
        expected_results = [12.41, 11.34, 1.30, 3.57, 0.09, 3.57]

        calculated_results = [s.aep_GWh.mean(),
                              s.aep_GWh.std() / s.aep_GWh.mean() * 100,
                              s.avail_pct.mean() * 100,
                              s.avail_pct.std() / s.avail_pct.mean() * 100,
                              s.curt_pct.mean() * 100,
                              s.curt_pct.std() / s.curt_pct.mean() * 100, ]

>       nptest.assert_array_almost_equal(expected_results, calculated_results, decimal=0)
E       AssertionError:
E       Arrays are not almost equal to 0 decimals
E
E       Mismatched elements: 1 / 6 (16.7%)
E       Max absolute difference: 2.59642721
E       Max relative difference: 0.46054407
E        x: array([12., 11.,  1.,  4.,  0.,  4.])
E        y: array([12., 11.,  1.,  5.,  0.,  6.])

test\regression\int_pruf_plant_analysis.py:239: AssertionError
------------------------------------------------ Captured stderr call -------------------------------------------------
100%|██████████| 30/30 [00:00<00:00, 103.15it/s]
```

**Comment by jordanperr** [comment]
Hi Charlie, I am getting the same error. Will look into this and get back to you.

**Comment by jordanperr** [comment]
It appears this regression test was not updated correctly in a recent pull request. I have started to fix it [here](https://github.com/jordanperr/OpenOA/tree/hotfix/regression-tests).

**Comment by jordanperr** [comment]
I have merged the fix for this issue into #132
